analysis/combiners.py

```diff
@@ -300,45 +340,46 @@ class CompoundCombiner(pipeline_dp.combiners.CompoundCombiner):
     # contribution to such accumulators leads to memory usage blow-up. That is
     # why sparse mode introduced - until the number of privacy id contributions
     # is small, they are saved instead of creating accumulators.
-    SparseAccumulatorType = List[PreaggregatedData]
+    SparseAccumulatorType = Tuple[List[int], List[float], List[int]]
     DenseAccumulatorType = List[Any]
     AccumulatorType = Tuple[Optional[SparseAccumulatorType],
                             Optional[DenseAccumulatorType]]
 
     def create_accumulator(self, data: PreaggregatedData) -> AccumulatorType:
         if not data:
-            # This is an empty partition, what is None assigned to. It was added
-            # because of public partitions.
-            return ([(0, 0, 0)], None)
-        return ([data], None)
+            # This is an empty partition. It was added because of public
+            # partitions.
```

Comment:
nit: Maybe something like  `# Handle empty partitions. Only encountered when public partitions are used.` could be better here